compiler/parser/parser.cpp

```diff
@@ -66,7 +67,8 @@ static ModuleSymbol* parseMod(const char* modName,
 
 static ModuleSymbol* parseFile(const char* fileName,
                                ModTag      modTag,
-                               bool        namedOnCommandLine);
+                               bool        namedOnCommandLine,
+                               bool        include);
```

Comment:
Probably not worth changing now, but I might've just made `include=false` by default to avoid needing to change most existing cases.